src/cdk/stepper/stepper.ts

```diff
@@ -63,6 +64,37 @@ export class StepperSelectionEvent {
   previouslySelectedStep: CdkStep;
 }
 
+/** The state of each step. */
+export type StepState = 'number' | 'edit' | 'done' | 'error' | string;
+
+/** Enum to represent the different states of the steps. */
+export const STEP_STATE = {
```

Comment:
enum name should be class-case, maybe `StepStates`?

Reply:
When I renamed the enum to `StepStates` I get an `tslint` error saying: 
```
[tslint] variable name must be in lowerCamelCase or UPPER_CASE (variable-name)
```

Reply:
In regards to the comment below, I cannot reference the enum in the `type` StepState since it is not of type `type` as well.